include/zephyr/bluetooth/audio/bap.h

```diff
@@ -1019,6 +1033,33 @@ struct bt_bap_unicast_server_cb {
 	int (*release)(struct bt_bap_stream *stream, struct bt_bap_ascs_rsp *rsp);
 };
 
+/**
+ * @brief Register the Unicast Server.
+ *
+ * Register the Unicast Server.
+ * This will register ASCS in the GATT database.
+ *
+ * @param param  Registration parameters for ascs.
+ *
+ * @return 0 in case of success.
+ */
+int bt_bap_unicast_server_register(const struct bt_bap_unicast_server_register_param *param);
+
+/**
+ * @brief Unregister the Unicast Server.
+ *
+ * Unregister the Unicast Server.
+ * This will unregister ASCS in the GATT database.
+ * This function will also unregister any functions registered with
+ * bt_bap_unicast_server_register_cb().
```

Comment:
Will it also release all ASEs? That is unclear from this description. 

It also seem to not match the implementation where it would return an error if they are registered, rather than unregistering them

Reply:
Yes, I indeed forgot to update this signature.